bip-0032.mediawiki

```diff
@@ -64,6 +65,8 @@ In what follows, we will define a function that derives a number of child keys f
 
 We represent an extended private key as (k, c), with k the normal private key, and c the chain code. An extended public key is represented as (K, c), with K = point(k) and c the chain code.
 
+When deriving child keys, a 'hardened' child key can only be generated using a private key. This provides security advantages but reduces the usefulness of HD key derivation. It is typically used to separate 'accounts' from each other.
```

Comment:
ACK this change.

Reply:
I'd prefer the last sentence be dropped. Given the weird security properties of "normal" derivation, I think we should word things as though hardened derivation were the ordinary case, and non-hardened derivation is the one that has a sentence explaining why somebody would want to use it.

Reply:
How about this wording to make it sound more ordinary:

"When deriving child keys, a 'hardened' child key can only be generated using a private key. This provides security advantages ***and prevents adversarial public key tracing***. It is typically used to separate 'accounts' from each other."

And another sentence added for non-hardened keys:

"***This non-hardened result is typically used by a server to continually generate receive addresses without the ability to spend funds.***"